Tests/iOS/Tests/Cache/SpecializedCacheTests.swift

```diff
@@ -33,6 +36,7 @@ final class SpecializedCacheTests: XCTestCase {
   // MARK: - Async caching
 
   func testAsyncAddObject() throws {
+    cache = SpecializedCache<User>(name: cacheName)
```

Comment:
Why did you move it here instead of `setup`?

Reply:
Had some issues with async calls from the `setup` method clearing the cache even if the test shouldn't do that.

Reply:
But I kinda don't want to leave it like this, the tests feel a bit icky now :/